src/components/editor/editor-pane/tool-bar/table-picker/table-picker.scss

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+.table-picker-container {
+  z-index: 1111;
+}
+
+div.table-cell {
+  border-top: 1px solid #000000;
+  border-left: 1px solid #000000;
+}
+
+div.table-container {
+  border-bottom: 1px solid #000000;
+  border-right: 1px solid #000000;
+}
```

Comment:
for better dark mode support this should use `$dark` instead. It's okay like it is now, but it could be better I think

Reply:
Depends on #802

Reply:
fixed